dashboard/building_dashboard.html

```diff
@@ -0,0 +1,256 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width">
+	<title>SF Building Dashboard</title>
+	<link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet-0.7.3/leaflet.css" />
+	<!-- Bootstrap core CSS -->
+	<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+	<!-- font awesome -->
+	<link rel= "stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css">
+
+	<link rel="stylesheet" href="css/building_dashboard.css">
+	<!-- <link rel="stylesheet" href="css/normalize.css"> -->
+	<!-- <link rel="stylesheet" href="css/style.css"> -->
+</head>
+<body>
+
+<div class="container-fluid">
+	<div class="row">
+	    <nav class="navbar navbar-default navbar-fixed-top">
+	      <div class="container">
+	        <div class="navbar-header">
+	          <a class="navbar-brand" href="../assets/sf_logo.png"></a>
+	        </div>
+	      </div>
+	    </nav>
+    </div>
+      <div class="row">
+      	<div class="col-xs-3">
+			<div class="row">
+			    <!-- <div class="panel panel-default"> -->
+	                <div class="panel-body">
+	                	<div class="col-xs-12">
+	                		<div class="row center">
+	                			<h5>BENCHMARKING PROFILE</h5>
+	                		</div>
+	                		<h2>Hotel Zetta</h2>
+	                		<div class="row center address">
+	                			<h4>55 5th Street<br>
+	                			San Francisco, CA</h4>
+	                		</div>
+	                		<div class="row center">
+	                			<div class="col-xs-6 building-units">
+	                				<div class="row">
+	                				Building type:
+	                				</div>
+	                				<div class="row">
+	                				Floor area:
+	                				</div>
+	                				<div class="row">
+	                				Compliance rate:
+	                				</div>
+	                			</div>
+	                			<div class="col-xs-6 building-data">
+	                				<div class="row">
+	         						Hotel 
+	                				</div>
+	                				<div class="row">
+	                				105,905 ft
+	                				</div>
+	                				<div class="row">
+	                				100%
+	                				</div>
+	                			</div>
+	                		</div>
+	                		<div class="row center">
+	                			<div class="side-panel-text">
+	                				2016 BENCHMARKS ARE DUE<br>
+	                				APRIL 1, 2017
+	                			</div>
+	                		</div>
+	                		<div class="row center">
+	                			<button type="button" class="btn btn-primary btn-round-lg btn-lg">SUBMIT NOW</button>
```

Comment:
should the button be a `<a href="http://sfenvironment.org/submit" class="btn btn-primary ...` instead, so clicking goes somewhere?